201-web-app-public-certificate/azuredeploy.json

```diff
@@ -0,0 +1,38 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "exisitngWebAppName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the web app that would use this public certificate. The web app should belong to a dedicated App Service Plan."
+            }
+        },
+        "publicCertificateLocation": {
+            "type": "string",
+            "allowedValues": ["CurrentUserMy", "LocalMachineMy"],
+            "metadata": {
+                "description": "Location where you wish to install the public certificate. 'LocalMachineMy' is supported in App Service Environment only."
+            }
+        },
+        "blob": {
+            "type": "string",
+            "metadata": {
+                "description": "Base 64 encoded public certificate file"
+            }
+        }
+    },
+    "variables": {
+        "publicCertificateName": "[concat(uniqueString(resourceGroup().id, deployment().name),'publicCertificate')]"
```

Comment:
What are the requirements for cert name that it needs to be unique and non-descriptive?  Should we just use a literal value instead?  or based on the web app name?  e.g. concat(parameters('existingwebappname'), '-cert')?